squape/object_tree.py

```diff
@@ -0,0 +1,273 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2023, Cyber Alpaca
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+import math
+import types
+
+try:
+    import squish
+except ImportError:
+    import squishtest as squish
+
+import object
+
+
+def children(object_name: any, selector: dict) -> tuple:
+    """
+    Finds direct children of the given object.
+
+    Args:
+        object_name (any): symbolic name, real name, or object reference to the parent.
+
+        selector (dict): The selector is a dictionary of key-value pairs,
+        where a key is a property of an object, and value is expected value.
+        The object will pass verification
+        if the object's property value matches the selector value.
+        Selectors also may include functions.
+        The passed functions must accept exactly 1 argument.
```

Comment:
```suggestion
        selector (dict): The selector is a dictionary of key-value pairs,
            where a key is a property of an object, and value is expected value.
            An object will pass verification
            if the object's property value matches the selector value.
            Selectors may include functions.
            The passed functions must accept exactly one argument.
```

Reply:
Ah, I misunderstood a bit. Now it is clear. Please see my last commit.